packages/flutter/test/widgets/editable_text_show_on_screen_test.dart

```diff
@@ -208,6 +225,9 @@ void main() {
     ));
 
     // Focus the EditableText and scroll it off screen.
+    scrollController.jumpTo(350.0);
```

Comment:
Did you change logic of the test intentionally?
If yes, why it is needed for leak coverage?
If it is side effect of git merge, let's clean it up.

Reply:
My bad. I missed the initial offset in the ScrollController. Reverted this one.